core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -42,6 +42,7 @@ object TransactionManager {
   // default transaction management config values
   val DefaultTransactionalIdExpirationMs = TimeUnit.DAYS.toMillis(7).toInt
   val DefaultTransactionsMaxTimeoutMs = TimeUnit.MINUTES.toMillis(15).toInt
+  val DefaultRemoveExpiredTransactionsIntervalMs = TimeUnit.MINUTES.toMillis(1).toInt
```

Comment:
I think we should add another broker config in `KafkaConfig` with default value to 1 minute, and read that value into `txnStateManager`?

For the other two values there are already configs in `KafkaConfig`, in this PR we can read in the `TransactionsMaxTimeoutMsProp` and replace the one in line 44.